os-neutron.te

```diff
@@ -80,6 +81,8 @@ corecmd_getattr_all_executables(neutron_t)
 # Bugzilla 1294420
 allow neutron_t radvd_exec_t:file getattr;
 
+# Allow execute execute_no_trans on neutron files
+allow neutron_t neutron_exec_t:file execute_no_trans;
```

Comment:
I would use this macro:
can_exec(neutron_t,neutron_exec_t)

Reply:
Thanks, updated as suggested.